GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -459,6 +459,65 @@ private extension CallVisualizer.Coordinator {
 
         return presenter
     }
+
+    func fetchSiteConfigurations(_ completion: @escaping (CoreSdkClient.Site) -> Void) {
+        environment.fetchSiteConfigurations { [weak self] result in
+            switch result {
+            case let .success(site):
+                completion(site)
+            case .failure:
+                self?.showUnexpectedErrorAlert()
+            }
+        }
+    }
+
+    func showSnackBarMessage() {
+        let style = environment.viewFactory.theme.snackBar
+        snackBar.present(
+            text: Localization.LiveObservation.Indicator.message,
```

Comment:
Is it possible to deliver  Localization.LiveObservation.Indicator.message via Theme? Or environment?

Reply:
Fixed 48c1d841778a2a04b4cd793006a83cbe779fa235